core/services/remoraCameraManager/remoraCameraManager.py

```diff
@@ -0,0 +1,495 @@
+import glob
+import json
+import os
+import re
+import select
+import shutil
+import stat
+import subprocess
+import textwrap
+from pathlib import Path
+from time import sleep, time
+from typing import Any, Dict
+
+from commonwealth.settings.manager import Manager
+from pykson import Pykson
+
+import usbPortControl
+from settings import CameraConfig, SettingsV1
+
+SERVICE_NAME = "remoraCameraManager"
+
+USERDATA = Path("/usr/blueos/userdata/")
+
+pyk = Pykson()
+default_config: Dict[str, Any] = {
+    "PORT": 8554,
+    "IP": "127.0.0.1",
+    "USE_HW_ENC": False,
+    "USB_HUB": "1-1",
+    "FRONT": {
+        "usb_hub_port": 2,
+        "device": "/dev/video4",
+        "resolution": "640x480",
+        "fps": 25,
+        "name": "front",
+        "kbitrate": 1000,
+        "preset": "superfast",
+    },
+    "BACK": {
+        "usb_hub_port": 1,
+        "device": "/dev/video0",
+        "resolution": "640x480",
+        "fps": 25,
+        "name": "back",
+        "kbitrate": 1300,
+        "preset": "ultrafast",
+    },
+}
+
+
+class RemoraCameraManager:
+    """Manager for config changes on remora camera controllers."""
+
+    def __init__(self) -> None:
+        self.settings_manager = Manager(SERVICE_NAME, SettingsV1, USERDATA / "settings" / SERVICE_NAME)
+        self.settings_manager.load()
+        if not self.settings_manager.settings.camera:
+            self.settings_manager.settings.camera = pyk.from_json(default_config, CameraConfig)
```

Comment:
Maybe I am confused by pykson, but I believe the default_config is not JSON, but a dict
```suggestion
            self.settings_manager.settings.camera = pyk.from_json(json.dumps(default_config), CameraConfig)
```